test/tomo/backends/astra_cpu_test.py

```diff
@@ -117,7 +115,6 @@ def test_astra_cpu_projector_fanflat():
                                                    dtype='float32')
 
     # forward
-    discr_data = odl.util.phantom.cuboid(discr_vol_space, 0.5, 1)
     proj_data = odl.tomo.astra_cpu_forward_projector_call(discr_data, geom,
                                                           discr_proj_space)
     assert proj_data.shape == proj_shape
```

Comment:
Cannot comment further below, doing it here.

The test `assert proj_data > 0` makes no sense at all and fails on Py3. Python 2 somehow makes sense of this comparison in some nonsense way.